benchmarks/llama.py

```diff
@@ -0,0 +1,236 @@
+import argparse
+import os
+import subprocess
+import json
+import datetime
+import re
+import sys
+
+
+def parse_args():
```

Comment:
nit: Have this above the main for consistency with the other scripts?

Reply:
Done.